src/features/dynamicRender/components/form/index.ts

```diff
@@ -0,0 +1,18 @@
+import { observer } from "lib/mobx";
+import { Checkbox } from "./Checkbox";
+import { registerComponent } from "features/dynamicRender/utils/registry";
+import { NumberInput } from "./NumberInput";
+import { TextInput } from "./TextInput";
+import { Radio } from "./Radio";
+import { TextArea } from "./TextArea";
+
+/**
+ * Registers the form components
+ */
+export function registerForm() {
```

Comment:
I kind of screwed up selecting the code block for this, I meant to show the whole function. This can apply to the other registerX() functions as well.